src/components/InformationDialog/InformationDialog.js

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { spacing } from '@ui5/webcomponents-react-base';
+import { Icon } from '@ui5/webcomponents-react/lib/Icon';
+import { Button } from '@ui5/webcomponents-react/lib/Button';
+import { ButtonDesign } from '@ui5/webcomponents-react/lib/ButtonDesign';
+import { Dialog } from '@ui5/webcomponents-react/lib/Dialog';
+import { FlexBox } from '@ui5/webcomponents-react/lib/FlexBox';
+import { FlexBoxDirection } from '@ui5/webcomponents-react/lib/FlexBoxDirection';
+import { FlexBoxAlignItems } from '@ui5/webcomponents-react/lib/FlexBoxAlignItems';
+import { FlexBoxJustifyContent } from '@ui5/webcomponents-react/lib/FlexBoxJustifyContent';
+import { TextWrapper } from '../../util/Layout'
+import Constants from '../../util/Constants';
+
+const _getHeaderIcon = type => {
+  switch (type) {
+    case Type.Warning:
+      return _getHeaderWarningIcon();
+    case Type.Error:
+      return _getHeaderErrorIcon();
+    default:
+      return _getHeaderInfoIcon();
+  }
+};
+
+const _getHeaderWarningIcon = () => {
+  return (
+    <Icon
+      name="message-warning"
+      style={{
+        width: '1.5rem',
+        height: '1.5rem',
+        color: '#feb60a'
```

Comment:
should we use CSS variables like `var(--sapWarningColor)` or similar?

Reply:
i think id would be good to refactor for that